app/mailers/app_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class AppMailer < ActionMailer::Base
+  def send_welcome_email(user)
+    @user = user
+    mail from: 'info@myflix.com', to: user.email, subject: "Welcome to myflix"
+  end
+
+  def send_password_reinitialization_email(user)
+    @user = user
+    mail from: 'info@myflix.com', to: user.email, subject: "Please reset your password"
```

Comment:
You can refactor `from: 'info@myflix.com'` to `default from: 'info@myflix.com'` and have it above methods. It should work the same as `before_action` in Controller.